tests/e2e/kcs/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,307 @@
+import logging
+import pytest
+import yaml
+import random
+import time
+import tempfile
+from datetime import datetime
+
+from ocs_ci.framework import config
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.ocs.exceptions import PodNotCreated
+from ocs_ci.ocs.resources import pod as res_pod
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility.templating import dump_data_to_temp_yaml
+
+
+log = logging.getLogger(__name__)
+
+
+class TestSelinuxrelabel(E2ETest):
+    def create_deploymentconfig_pod(self, **kwargs):
+        """
+        create deployment pod.
+
+        """
+
+        # Create service_account to get privilege for deployment pods
+        self.sa_name = helpers.create_serviceaccount(namespace=self.project_namespace)
```

Comment:
you can use service_account_factory which will take care of teardown code as well